eval.go

```diff
@@ -292,6 +292,10 @@ type sortByOffsetSlice []*candidateMatch
 func (m sortByOffsetSlice) Len() int      { return len(m) }
 func (m sortByOffsetSlice) Swap(i, j int) { m[i], m[j] = m[j], m[i] }
 func (m sortByOffsetSlice) Less(i, j int) bool {
+	if m[i].scope == query.ScopeSymbol {
+		return m[i].file < m[j].file
```

Comment:
why is the data stored this way, instead of it being like content and filenames where everything is concatted together so we can just use byteOffsets.